src/pages/profile/profile.rs

```
@@ -192,208 +182,177 @@ pub fn Profile(cx: Scope) -> Element {
+                                            if !original_profile.read().displayname.eq(&current_profile.read().displayname) {
+                                                let x = client.get().account().set_display_name(Some(current_profile.read().displayname.as_str())).await;
+                                                info!("{x:?}");
+                                                match x {
+                                                    Ok(_)=> {}
+                                                    Err(_)=> {}
+                                                }
                                             }
-                                        }
 
-                                        if let Some(y) = attach.get() {
-                                            let x = client.get().account().upload_avatar(&mime::IMAGE_PNG, &y.data).await;
-                                            info!("{x:?}");
-                                            match x {
-                                                Ok(url)=> {
-                                                    let x = client.get().account().set_avatar_url(Some(&url)).await;
-                                                    info!("{x:?}");
+                                            if let Some(y) = attach.get() {
+                                                let x = client.get().account().upload_avatar(&mime::IMAGE_PNG, &y.data).await;
+                                                info!("{x:?}");
+                                                match x {
+                                                    Ok(url)=> {
+                                                        let x = client.get().account().set_avatar_url(Some(&url)).await;
+                                                        info!("{x:?}");
+                                                    }
+                                                    Err(_)=> {}
```

Same thing with the error ... Rust complains about unused results for a reason, ignoring errors is a recipe for disaster